Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "qdrant-client"
-version = "1.6.0"
+version = "1.7.0"
```

Comment:
Not 1.8?

Reply:
I had the impression the actual version update is done when pushing to master.

But I can change it if it is not the case :)

Reply:
decided to fix it right away!